frontend/packages/dev-console/src/components/pipelineruns/PipelineRunRow.tsx

```diff
@@ -45,7 +48,7 @@ const PipelineRunRow: React.FC<PipelineRunRowProps> = ({ obj, index, key, style
       <TableData className={tableColumnClasses[3]}>
         <LinkedPipelineRunTaskStatus pipelineRun={obj} />
       </TableData>
-      <TableData className={tableColumnClasses[4]}>-</TableData>
+      <TableData className={tableColumnClasses[4]}>{pipelineRunDuration(obj)}</TableData>
```

Comment:
@abhinandan13jan Is there any reason for not using [fromNow](https://github.com/openshift/console/blob/master/frontend/public/components/utils/datetime.ts#L4) function?

Reply:
@vikram-raj  If the Pipeline is currently running, `fromNow` would be applicable.
But for Pipelines which have `completed/failed` at a past timestamp. the duration must be the `end-start` time and not `fromNow`

Reply:
@abhinandan13jan I did not notice that we are using duration for the pipeline.